### PR TITLE
resource_control: fix resource control limitation

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15121
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
@nolouch PTAL

File: tidb-resource-control.md [hunk 1]
```diff
@@ -36,10 +36,7 @@ In addition, the rational use of the resource control feature can reduce the num
 
 ## Limitations
 
-Currently, the resource control feature has the following limitations:
-
-* This feature only supports flow control and scheduling of read and write requests initiated by foreground clients. It does not support flow control and scheduling of background tasks such as DDL operations and auto analyze.
-* Resource control incurs additional scheduling overhead. Therefore, there might be a slight performance degradation when this feature is enabled.
+Resource control incurs additional scheduling overhead(about 5%). Therefore, there might be a slight performance degradation when this feature is enabled.
```
Comment (Oreoxmt) [inline tidb-resource-control.md:39]:
```suggestion
Resource control incurs additional scheduling overhead (about 5%). Therefore, there might be a slight performance degradation when this feature is enabled.
```

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: tidb-resource-control.md [hunk 2]
```diff
@@ -36,10 +36,7 @@ In addition, the rational use of the resource control feature can reduce the num
 
 ## Limitations
 
-Currently, the resource control feature has the following limitations:
-
-* This feature only supports flow control and scheduling of read and write requests initiated by foreground clients. It does not support flow control and scheduling of background tasks such as DDL operations and auto analyze.
-* Resource control incurs additional scheduling overhead. Therefore, there might be a slight performance degradation when this feature is enabled.
+Resource control incurs additional scheduling overhead (about 5%). Therefore, there might be a slight performance degradation when this feature is enabled.
```
Comment (Oreoxmt) [inline tidb-resource-control.md:39]:
```suggestion
Resource control incurs additional scheduling overhead. Therefore, there might be a slight performance degradation (less than 5%) when this feature is enabled.
```

**Review by nolouch — APPROVED** [review]

**Pull request merged**